### PR TITLE
Platui 542 cookie banner feature toggle

### PR DESCRIPTION
**Pull request by JoPintoPaul**

### CODE REVIEW

File: app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Cookies from 'js-cookie'
+
+const isFeatureEnabled = (featureName: String) => {
+    function getEnabledFlagFromCookie(featureName: String) {
```
Comment (matthewmascord) [inline app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts:4]: Could use arrow function const getEnabledFlatFromCookie = ... as above?

File: app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        return (Cookies.get(featureName) == 'true')
+    }
+
+    function getEnabledFlagFromQuery(featureName: String) {
+        return window.location.search.includes(featureName.concat('=true'))
```
Comment (matthewmascord) [inline app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts:9]: Would an ES6 template literal be easier to read here vs .concat?

File: app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts (hunk 1 above)
Comment (JoPintoPaul) [reply app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts]: Done.

File: app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts (hunk 2 above)
Comment (JoPintoPaul) [reply app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts]: Done.

**Pull request merged**